### ISSUE TITLE
Browser freezes when trying to open ResqueBoard

### ISSUE THREAD

**Issue by jstuhli** [wa0x6e/ResqueBoard#16]
After a clean install, when trying to open the ResqueBoard main page, when all of the resources are loaded, the browser freezes, and uses 100% CPU. After a time Chrome asks whether to kill the page.

Tested on: Chrome and Firefox
IE doesn't freeze but it also doesn't load anything (shows empty dashboard without any graphs or loaders).

**Comment by wa0x6e** [comment]
> when all of the resources are loaded

So you see everything normally, but graph and numbers does not "blink" ?
Did you try opening other page instead ? Pages like /logs/browse, /jobs/view, /jobs/distribution/load doesn't use websocket, and are pretty much static.

Is there activity on the Resque end ?
What's your timezone ? I suspect the freeze to be due to a lot of data incoming via websocket, due to javascript not fetching more data than needed, because of a timezone offset.

**Comment by jstuhli** [comment]
![resque_board](https://f.cloud.github.com/assets/1474280/863419/22630cbc-f609-11e2-907d-822d41e65afe.png)

I've attached a screenshot, the 3 circular gifs are animated, everything else is static. I cannot click anywhere. My timezone is CET (Central European Time +1).

I've found the problem since I started writing this comment:
The problem was in my nginx config file: all other routes (except for static files and /) were not property served to the client (every route served content from /), it seems that that's what caused the freeze.
